jfix-stdlib-concurrency/src/test/java/ru/fix/stdlib/concurrency/futures/PendingFutureLimiterTest.java

```diff
@@ -282,6 +283,26 @@ public void waitAll_releases_thread_after_last_future_is_completed() throws Exce
         assertTimeoutPreemptively(assertingTaskTimeout, () -> limiter.waitAll(largeTimeout));
     }
 
+    @Test
+    public void enqueueBlocking_blocks_invoking_thread_when_limit_is_reached() throws Exception {
+        PendingFutureLimiter limiter = new LimiterBuilder()
+                .executionTimeLimit(0)
+                .enqueueTasks(3)
+                .build();
+        AtomicBoolean ai = new AtomicBoolean(false);
+        // We'll try to enqueue after futures timeout is exceeded
+        Executors.newSingleThreadScheduledExecutor().execute(() -> {
+            try {
+                limiter.enqueueBlocking(createTask());
```

Comment:
We can see that maxPendingCount is 3 by default, that makes test confusing since we have to go deep into implementation of the builder to understand that magic 3  is actually maxPendingCount. 
please, make it explicit: 
```
.executionTimeLimit(3)
.maxPendingCount(3)
```

Reply:
Fixed this uncertainity in some other way.

Reply:
buildAndFulyEnqueue() still does not show maxPendingCount size to the reader of the test. 
What if it is 0, then what is a meaning of  buildAndFulyEnqueue()? 

maxPendingCount is a main property of the limiter, hiding it behind builder internals hides main aspect of the limiter class. Limiter is all about maxPendingCount so could you please make this configuration explicit.

Reply:
Fixed, made it explicit.